spec/models/library_entry_spec.rb

```diff
@@ -133,6 +133,52 @@
     end
   end
 
+  describe 'timestamp validation' do
```

Comment:
Yeah I have to agree, `let!(:library_entry)` is better than using `@library_entry` in `before`

Both `let` and `let!` provide memoized "variables", but `let` will initialize on first usage while `let!` will be pre-initialized before the specs run